src/data-table.js

```diff
@@ -310,7 +314,14 @@ export class DataTable {
         if (c.type === COL_TYPES.T_CLEAN) {
           col.Cell = ({ cell: { value } }) => <TimeWithTooltip timestamp={value} />;
         }
-        return col;
+  
+      // Takes out the value from the cell if the cell
+      // is in a region (Currently solves the uppercase issue).
+      if(c.type === COL_TYPES.STATE){
+        col.Cell = ({ cell: { value } }) => value;
+      }
+
+      return col;
```

Comment:
Good job for figuring out you had to do this! I still cannot tell you why it works with this and doesn't work without it :)

Can you also check if `c.type === COL_TYPES.STATE_TMP`? That will make the capitalization also work for the "temporary" state column that comes up when you are in the middle of creating a region.

Also: the indentation for these lines is off, if you can fix that :)